deployment/6_register_modules.js

```diff
@@ -43,24 +45,44 @@ const deploy = async (network) => {
   const NftTransferWrapper = await deployer.wrapDeployedContract(NftTransfer, config.modules.NftTransfer);
   const CompoundManagerWrapper = await deployer.wrapDeployedContract(CompoundManager, config.modules.CompoundManager);
   const MakerV2ManagerWrapper = await deployer.wrapDeployedContract(MakerV2Manager, config.modules.MakerV2Manager);
+  const RelayerManagerWrapper = await deployer.wrapDeployedContract(RelayerManager, config.modules.RelayerManager);
+  const VersionManagerWrapper = await deployer.wrapDeployedContract(VersionManager, config.modules.VersionManager);
 
   const ModuleRegistryWrapper = await deployer.wrapDeployedContract(ModuleRegistry, config.contracts.ModuleRegistry);
   const MultiSigWrapper = await deployer.wrapDeployedContract(MultiSig, config.contracts.MultiSigWallet);
 
-  const wrappers = [
-    GuardianManagerWrapper,
-    LockManagerWrapper,
-    RecoveryManagerWrapper,
-    ApprovedTransferWrapper,
-    TransferManagerWrapper,
-    TokenExchangerWrapper,
-    NftTransferWrapper,
-    CompoundManagerWrapper,
-    MakerV2ManagerWrapper,
+  const wrappers = [VersionManagerWrapper];
+
+  // Add Features to Version Manager
+  const newFeatures = [
```

Comment:
`newFeatures` -> `features`

Reply:
Thanks, yes and I corrected a few other instances in the same script referring to `newFeatures`